### ISSUE TITLE
[CAIA A11y Research]: Ask VA VA.gov Team, Ask VA - Screen Magnification Prototype (April & Late Aug. 2024)

### ISSUE THREAD

**Issue by tygindraux** [department-of-veterans-affairs/va.gov-team#80535]
> [!IMPORTANT]
> ## Late August Research Ask VA
> - **Research Timeframe**: `LATE AUGUST`
> - **Time of Day:** `TBD`
> - **_Future_ Pilot**:  `Mobile Pilot` & `Desktop Pilot` 
> - **_Future_ Assistive Tech Users**: `TBD #` AT Users
> - **_Future_ Testing Environment:** `CodePen or Staging`
> - **_Future_ Research Assistive Tech Being Used**:  `screen magnification` and `screen readers` 
> - **Recent Research:** #82856

> [!NOTE]
> ## Timing Update 
> While we’re waiting on everything to be ready in staging, we’ll probably sneak in one more research study using Figma. I’m guessing we’re looking at ~July~ `AUGUST` for next steps testing with screen mag and screen reader users. - c/o Tyler 5/10/2024 (UPDATED 5/14/2024 PER JAMIE)

<hr>

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- @coforma-jamie (pilot)
- @sara-amanda 
- `TBD`

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- **Team name:** Ask VA VA.gov
- **OCTO-DE product owner:** Becky Phung
- **Product manager:** Holly Hansen
- **Designer:** Tyler Gindraux, Thomas Michaud
- **FE engineer:** Hemesh Patel, Eddie Otero
- **Product/team Slack channel:** #ask-va-public

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe
</details>

## Future Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **Pilot Session**:`4/30/2024 @ 10 a.m.`
- **Timeframe**: `TBD`

 > "For awareness, we're planning to run an AT round once we have work ready in staging. We're not there yet (and we don't have an estimated date yet), but I'll reach out to you as soon as I have a rough date for that research study. We would plan to include SR and screen mag users then."

### Length of sessions

- **Session length:** `pending` minutes
- **Buffer time between sessions:** `pending` minutes
- **Maximum sessions per day:** `pending`

</details>

##  Future Testing Schedule

> [!IMPORTANT]  
> ### COMPLETED
> - Tyler will run a screen magnification pilot with a CAIA a11y using the team's Figma prototypes.
> - The team will run its study without AT users, and without CAIA support.
> - SEE TICKET #82856
> ### Later
> - The team will create a coded prototype in Codepen or Staging.
> - Tyler will run two screen reader pilots with CAIA a11ys - one on mobile, one on desktop.
> - The team will run a new study with AT users - magnification and screen reader - with CAIA support.
> - This ticket #81692 only covers the screen magnifier "NOW" pilot.

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
`PILOT` `TESTING 1` | 4/30/24, 10 a.m. | screen magnification | Desktop; MacOS | Safari | Advanced | @coforma-jamie 

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/products/ask-va-phase-1-product-outline.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708479170047/8ebb517a213e05ebc500087f5661a993a41392e8?wid=1358-1710439493448)
- [Prototype or mockups](https://www.figma.com/file/SYvmqLiAKaGtIcePSRRimB/Ask-VA-Prototype-R2?type=design&mode=design&t=hk9eNPHEIgYj7Jy9-0)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/04-2024%20Progress%20bar/Research%20plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/04-2024%20Progress%20bar/Conversation%20guide.md)
- [Epics/issue that may be helpful](https://github.com/orgs/department-of-veterans-affairs/projects/929/views/1?filterQuery=ask+va&pane=issue&itemId=31505793)

</details>

**Comment by coforma-jamie** [comment]
After discussing the team's needs with @tygindraux, we determined the following:

### Now
- Tyler will run a screen magnification pilot with a CAIA a11y using the team's Figma prototypes. 
- The team will run its study without AT users, and without CAIA support.

### Later
- The team will create a coded prototype in Codepen or Staging.
- Tyler will run two screen reader pilots with CAIA a11ys - one on mobile, one on desktop.
- The team will run a new study with AT users - magnification and screen reader - with CAIA support.
- **The team will need to create a new research ticket for this work. This ticket only covers the screen magnifier "NOW" pilot.**

**Comment by sara-amanda** [comment]
## Closing Ticket

- [x] Closing out this ticket that was used for Screen Magnification (only) a pilot was run by Jamie. 
- [x] Due to the shift in research plans, this ticket was also used to capture future planning for the August Research. 
- [x] New ticket received for the August research. #87480
